docs/contribute/code.md

```diff
@@ -59,7 +59,7 @@ yarn build
 When asked if you want to start the build in `watch` mode, answer **yes** to develop in interactive mode. Afterward, choose which packages you want to build. For example, if you're going to work on a feature for `@storybook/addon-docs`, you might want to select `@storybook/addon-docs` and `@storybook/components`.
 
 <div class="aside">
-💡 Build's `watch' mode is great for interactive development. However, for performance reasons it only transpiles your code and doesn't execute the TypeScript compiler. If something isn't working as expected, try running `build` <b>WITHOUT</b> watch mode: it will re-generate TypeScript types and also perform type checking for you.
```

Comment:
I believe we need to use the `code` tag here because the paragraph is in an HTML `div`.

Syntax highlighting doesn't work when I tried replacing the single quote with a backtick

| With `<code>` | With backtick |
| --- | --- |
| <img width="1127" alt="Screen Shot 2022-02-07 at 22 46 33" src="https://user-images.githubusercontent.com/25715018/152822898-afbffeea-5437-41a5-a713-8fec7b5844b4.png"> | <img width="1133" alt="Screen Shot 2022-02-07 at 22 54 21" src="https://user-images.githubusercontent.com/25715018/152823388-f2a01749-2902-4fc1-9e91-75d78c844f2a.png"> |

Reply:
@huyenltnguyen just a friendly reminder to see if you're willing to work on the feedback that @yannbf mentioned. And to give a bit more context on this if you apply the suggestion and add lines between the divs and text inside it will render properly, as follows:

```md

<div class="aside">

💡 Build's `watch` mode is great for interactive development. However, for performance reasons it only transpiles your code and doesn't execute the TypeScript compiler. If something isn't working as expected, try running `build` <b>WITHOUT</b> watch mode: it will re-generate TypeScript types and also perform type checking for you.

</div>

```

Let me know once you've addressed the feedback and we'll go from there.